flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -253,6 +253,13 @@
 				.noDefaultValue()
 				.withDescription("Specify YARN node label for the YARN application.");
 
+	public static final ConfigOption<String> PRE_UPLOADED_FLINK_PATH =
+		key("yarn.pre-uploaded.flink.path")
+			.stringType()
+			.noDefaultValue()
+			.withDescription("Specify the pre-uploaded path of flink. It will be used to accelerate the job submission" +
```

Comment:
```
Specifies the pre-uploaded path of the Flink Jar. If specified, Flink will locate the pre-uploaded Flink Jar via the provided path during container localization. This could accelerate the job submission process.
```

Reply:
I will change the description.